src/utils.ts

```diff
@@ -1,23 +1,48 @@
+import BigNumber from 'bignumber.js'
 import { createObjectCsvWriter } from 'csv-writer'
 import { ObjectStringifierHeader } from 'csv-writer/src/lib/record'
 import { existsSync, mkdirSync, writeFileSync } from 'fs'
 import 'isomorphic-fetch'
+import { Tags } from './entities/Tags'
 import { TransactionParsed } from './export-transactions'
-import { Network } from './interfaces/Network'
+import { KPI } from './interfaces/KPIs'
 import { TransactionDetails } from './interfaces/Transactions/Transactions'
 import { TransferType } from './interfaces/Transactions/Transfers'
 
-export const wallets = [
-  [Network.ETHEREUM, "0x9a6ebe7e2a7722f8200d0ffb63a1f6406a0d7dce", "Aragon Agent"],
-  [Network.ETHEREUM, "0x89214c8ca9a49e60a3bfa8e00544f384c93719b1", "DAO Committee"],
-  [Network.POLYGON, "0xb08e3e7cc815213304d884c88ca476ebc50eaab2", "DAO Committee"],
-]
-
 require('dotenv').config()
 
 export const COVALENT_API_KEY = process.env.COVALENTHQ_API_KEY
 export const INFURA_URL = process.env.INFURA_URL
 
+export function sum(array: number[]) {
+  return array.reduce((prev, curr) => prev + curr, 0)
+}
+
+export function avg(array: number[]) {
+  return sum(array) / array.length
+}
+
+export function median(array: number[]) {
+  if (array.length === 0) throw new Error("Median: no inputs")
+
+  array.sort((a, b) => a - b)
```

Comment:
You are modifying the original array here. You can copy it and store it in a new variable.
Maybe doing something like `const sortedArray = [...array].sort(...)`